tensorflow/lite/micro/kernels/xtensa/depthwise_conv_vision.cc

```diff
@@ -0,0 +1,135 @@
+#if defined(VISIONP6)
```

Comment:
Missing copyright info

Please run clang-format and cpplint.py on modified/added files before submitting https://github.com/tensorflow/tflite-micro/blob/main/CONTRIBUTING.md#before-submitting-your-pr

Reply:
I have pushed a commit with these fixes.